relay-general/src/store/transactions/processor.rs

```diff
@@ -438,12 +444,45 @@ impl Processor for TransactionsProcessor<'_> {
 
         span.op.get_or_insert_with(|| "default".to_owned());
 
+        if self.scrub_span_descriptions {
+            scrub_span_description(span)?;
+        }
+
         span.process_child_values(self, state)?;
 
         Ok(())
     }
 }
 
+fn scrub_span_description(span: &mut Span) -> Result<(), ProcessingAction> {
+    if span.description.value().is_none() {
+        return Ok(());
+    }
+
+    // let mut scrubbed: Annotated<Value::String> = span.description.clone();
+    let mut scrubbed = span.description.clone();
+
+    // Scrub URLs
+    if scrub_identifiers(&mut scrubbed)? {
+        let Some(new_desc) = scrubbed.value() else {
+            return Ok(());
+        };
+        span.data
+            .value_mut()
+            .as_mut()
+            // We don't care what the cause of scrubbing was, since we assume
+            // that after scrubbing the value is sanitized.
+            .and_then(|data| {
+                data.insert(
+                    "description.scrubbed".to_owned(),
+                    Annotated::new(Value::String(new_desc.to_owned())),
```

Comment:
I was having some issues to add the scrubbed description as the value here, and the dance around scrubbed values above is a result of this.